code-editor/auto-stop-idle/README.md

```diff
@@ -50,6 +50,45 @@ aws sagemaker update-domain \
     }'
 
 ```
+## Steps for installing in Internet-Free VPC environments:
+
+* Note : Your domain execution role must have privileges to download the packages from your s3 bucket where the below packages are uploaded. 
+
+1. Downloading autostop idle Python package and upload to s3 bucket.
+```
+ASI_VERSION=0.3.0
```

Comment:
Should this be placeholder for customer to replace with the latest version?

Reply:
Yes.